genestack_client/scripts/genestack_user_setup.py

```diff
@@ -202,6 +233,27 @@ def run(self):
         print 'Password has been changed successfully'
 
 
+class SetToken(Command):
+    COMMAND = 'token'
```

Comment:
I think that we should try to make all command _verbs_: `add`, `rename` etc.
Therefore I propose `COMMAND = 'SetToken'`

Reply:
Lets discuss it in person. 

Agenda:
- we have case sensitive command names
- we use only one word names for commands
- we already use `password` to change password